README.md

```diff
@@ -419,197 +745,512 @@ const myToolbar = {
+
+```js
+<DocumentView
+  onBehaviorActivated = {({action, data}) => {
+    console.log('Activated action is', action);
+    if (action === Config.Actions.linkPress) {
+      console.log('The external link pressed is', data.url);
+    }
+  }}
+/>
+```
 
 ##### pageChangeOnTap
 bool, optional, default to true
+
+Defines whether page should change on tap when viewer is in horizontal viewing mode
+
+```js
+<DocumentView
+  pageChangeOnTap={true}
+/>
+```
+
 ##### useStylusAsPen
 bool, optional, default to true
 
-If true, stylus will act as a pen in pan mode, otherwise it will act as finger
+Defines whether stylus should act as a pen in pan mode. If false, it will act as a finger.
+
+```js
+<DocumentView
+  useStylusAsPen={true}
+/>
+```
 
 ##### multiTabEnabled
 bool, optional, default to false
 
-If true, viewer will show multiple tabs for documents opened.
+Defines whether viewer will show tabs for documents opened. Changing the [`document`](#document) prop value will cause a new tab to be opened with the associated file.
+
+```js
+<DocumentView
+  multiTabEnabled={true}
+/>
+```
 
 ##### tabTitle
-string, optional, default to file name, takes effect when multiTabEnabled is true.
+string, optional, default to file name
+
+Set the tab title if [`multiTabEnabled`](#multiTabEnabled) is true.
+
+```js
+<DocumentView
+  multiTabEnabled={true} // requirement
+  tabTitle={'tab1'}
+/>
+```
 
 ##### signSignatureFieldsWithStamps
 bool, optional, default to false
 
-If true, signature field will be signed with image stamp.
+Defines wether signature field will be signed with image stamp.
 This is useful if you are saving XFDF to remote source.
 
+```js
+<DocumentView
+  signSignatureFieldsWithStamps={true}
+/>
+```
+
 ##### followSystemDarkMode
 bool, optional, Android only, default to true
 
-If true, UI will appear in dark color when System is dark mode. Otherwise it will use viewer setting instead.
+Defines whether UI will appear in dark color when System is dark mode. Otherwise it will use viewer setting instead.
+
+```js
+<DocumentView
+  signSignatureFieldsWithStamps={false}
+/>
+```
+
 ##### collabEnabled
-bool, optional, if set to true then `currentUser` must be set as well for collaboration mode to work
+bool, optional, default to false
+
+Defines whether to use the collaboration mode. If true then `currentUser` must be set as well for collaboration mode to work
+
+```js
+<DocumentView
+  collabEnabled={true}
+  currentUser={'Pdftron'}
+/>
+```
+
 ##### currentUser
-string, required if `collabEnabled` is set to true
+string, required if [`collabEnabled`](#collabEnabled) is set to true
+
+Defines the current user. Annotations created will has its title matching this string.
+
+```js
+<DocumentView
+  collabEnabled={true}
+  currentUser={'Pdftron'}
+/>
+```
+
 ##### currentUserName
 string, optional
+
+Defines the current user name. Will set the user name only if [`collabEnabled`](#collabEnabled) is true and [`currentUser`](#currentUser) is defined.
```

Comment:
```suggestion
Defines the current user name. Will set the user name only if [`collabEnabled`](#collabEnabled) is true and [`currentUser`](#currentUser) is defined. This should be used only if you want the user's display name to be different than the annotation's title/author (in the case that `currentUser` is an ID rather than a human-friendly name.)
```
@sgong-pdftron please confirm.